packages/local_auth/pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: local_auth
+description: Local authentication plugin for Flutter.
```

Comment:
Yep will do. The naming of this plugin is actually very tricky. On iOS local authentication is: https://developer.apple.com/documentation/localauthentication. However on Android such a thing is not possible. You have to do everything manually. For instance we have no PIN support. Hopefully android will improve its API and we can get something closer to iOS.